integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/KmAuthHealthIndicator.java

```diff
@@ -0,0 +1,38 @@
+package org.symphonyoss.integration.healthcheck.services;
+
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * Service health indicator for Key Manager Authentication.
+ *
+ * Created by hamitay on 30/10/17.
+ */
+@Component
+@Lazy
+public class KmAuthHealthIndicator extends AuthenticationServiceHealthIndicator {
+
+  private static final String SERVICE_NAME = "Key Manager Authentication Service";
+
+  private static final String SERVICE_FIELD = "keyauth";
+
+  @Override
+  protected String getServiceName() {
+    return SERVICE_NAME;
+  }
+
+  @Override
+  protected String getMinVersion() { return properties.getKeyManagerAuth().getMinVersion(); }
```

Comment:
You should return the minimum version just when the current version has already been read from POD.

See: https://github.com/symphonyoss/App-Integrations-Core/blob/dev/integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/PodSessionManagerHealthIndicator.java#L42